NEWS.md

```diff
@@ -1,5 +1,32 @@
 # What's new in `IntervalArithmetic.jl`
 
+## v0.13
+
+### Breaking changes
+- `convert(Interval{T}, x::T)` no longer does any rounding: it creates
+the thin (zero-width) interval `[x, x]`. [#114](https://github.com/JuliaIntervals/IntervalArithmetic.jl/pull/114)
+
+  Note that this implies that numbers are no longer rounded in operations combining them with intervals, such as `0.1 + interval(0.2, 0.3)`, since `0.1` is `convert`ed to an `Interval` by Julia's promotion machinery.
+
+### New functions
+- `atomic(Interval{T}, x)` creates the smallest atomic interval obtained by directed rounding, treating `x` as representing a real number (the old behaviour
```

Comment:
Since `atomic` is not exported, perhaps it is a good idea to indicate this, or write `IntervalArithmetics.atomic`...